web_src/js/components/ContextPopup.vue

```diff
@@ -3,16 +3,17 @@ import {SvgIcon} from '../svg.ts';
 import {GET} from '../modules/fetch.ts';
 import {getIssueColor, getIssueIcon} from '../features/issue.ts';
 import {computed, onMounted, shallowRef} from 'vue';
+import type {Issue} from '../types.ts';
 
 const props = defineProps<{
   repoLink: string,
   loadIssueInfoUrl: string,
 }>();
 
-const loading = shallowRef(false);
-const issue = shallowRef(null);
-const renderedLabels = shallowRef('');
-const errorMessage = shallowRef(null);
+const loading = shallowRef<boolean>(false);
+const issue = shallowRef<Issue>(null);
+const renderedLabels = shallowRef<string>('');
+const errorMessage = shallowRef<string>(null);
```

Comment:
* Does it need to repeat the `<type>` since `false` and `''` already have the type?
* When using `shallowRef<string>`, the value should be `''` but not `null`?

Reply:
1. it seems the rule does not trigger for boolean/string default values, so type it could be omitted there.
2. yes, last case could be altered to `errorMessage = shallowRef('');` probably.